src/constants.jl

```diff
@@ -186,33 +195,101 @@ const ONEWAY_FALSE = Set(["false", "no", "0", 0])
 """
 Default factor applied to maxspeed when the `lane_efficiency` weight is used to contruct OSMGraph object.
 """
-const LANE_EFFICIENCY = Dict(
+const LANE_EFFICIENCY = Ref(Dict{DEFAULT_OSM_LANES_TYPE,DEFAULT_OSM_EDGE_WEIGHT_TYPE}(
     1 => 0.7,
     2 => 0.8,
     3 => 0.9,
     4 => 1.0
-)
-
-"""
-Default data types used to construct OSMGraph object.
-"""
-const DEFAULT_OSM_ID_TYPE = Int64
-const DEFAULT_OSM_INDEX_TYPE = Int32
-const DEFAULT_OSM_EDGE_WEIGHT_TYPE = Float64
-const DEFAULT_OSM_MAXSPEED_TYPE = Int16
-const DEFAULT_OSM_LANES_TYPE = Int8
+))
 
 """
 Default height of buildings in metres.
 """
-const DEFAULT_BUILDING_HEIGHT_PER_LEVEL = 4
+DEFAULT_BUILDING_HEIGHT_PER_LEVEL = Ref{Real}(4)
```

Comment:
```suggestion
const DEFAULT_BUILDING_HEIGHT_PER_LEVEL = Ref{Float64}(4)
```
I'd keep the `const` here - it refers to the type of the variable not the value of the variable - and best to use a concrete type for type stability where `DEFAULT_BUILDING_HEIGHT_PER_LEVEL` is used.

And same thing for `DEFAULT_MAX_BUILDING_LEVELS`

Reply:
Good catch, `const` was removed by accident here.